keyboards/kbdfans/kbdpad/mk2/mk2.c

```diff
@@ -26,7 +26,7 @@ void matrix_init_kb(void) {
 }
 
 void led_set_kb(uint8_t usb_led) {
```

Comment:
Can be changed to `led_update_kb()`
Also `matrix_scan_kb()` and `process_record_kb()` can be deleted.